Sources/Kommunicate/Classes/KMAppSettingsService.swift

```diff
@@ -69,6 +69,8 @@ class KMAppSettingService {
        appSettings.buttonPrimaryColor = primaryColor
        appSettings.showPoweredBy = chatWidget.showPoweredBy ?? false
        appSettings.hidePostCTAEnabled = chatWidget.hidePostCTAEnabled ?? false
+        appSettings.defaultUploadOverrideUrl = chatWidget.defaultUploadOverride?.url ?? ""
```

Comment:
Formatting and extra lines in above change.

Reply:
resolved